### PR TITLE
[12.0][MIG] website_media_size: Migration to v12

### PR DESCRIPTION
**Pull request by keylor2906**
- Improve adding state colors depending on file size

### CODE REVIEW

**Review by tbaden — CHANGES_REQUESTED** [review]
please use the fragmented readme.
for more [information](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0)

File: website_media_size/static/src/scss/website_media_size.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        cursor: pointer;
+        font-size: 80%;
+    }
+    .o_existing_attachment_size_warning {
+        background-color: rgba(255, 213, 0, 0.8);
+    }
+    .o_existing_attachment_size_danger {
+        background-color: rgba(204, 0, 0, 0.8);
+    }
+}
```
Comment (tbaden) [inline website_media_size/static/src/scss/website_media_size.scss]: missing newline

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
please keep the commit history. For more info see [this](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0)

**Comment by oscarolar** [comment]
@keylor2906 can you please address the comments of this PR?

File: website_media_size/static/src/scss/website_media_size.scss (hunk 1 above)
Comment (keylor2906) [reply website_media_size/static/src/scss/website_media_size.scss]: Added line at EOF

**Comment by keylor2906** [comment]
Added the fragmented readme and keep the commit history

**Review by oscarolar — APPROVED** [review]
Tested in runbot, works as expected 
image

**Review by tbaden — APPROVED** [review]

**Pull request merged**